README.md

```diff
@@ -109,7 +109,7 @@ Flow is written in OCaml (OCaml 4.14.0 is required).
 6. Build `flow.js` (optional):
 
     ```sh
-    opam install -y js_of_ocaml.3.9.0
+    opam install -y js_of_ocaml.4.1.0
```

Comment:
Let's make it 4.0.0 instead. That's the version currently in the Makefile.

Reply:
That's fine by me. All I know is that 3.9 didn't work :)